CMakeLists.txt

```diff
@@ -38,6 +49,6 @@ target_include_directories(testrunner PRIVATE src test)
 
 target_compile_options(testrunner PRIVATE -Wextra -pedantic)
 
-target_link_libraries(testrunner PRIVATE gtest gtest_main ${CMAKE_THREAD_LIBS_INIT})
+target_link_libraries(testrunner PRIVATE ${CONAN_LIBS} ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
Are you sure that the last  var is still required?

Reply:
@malirod, removed